.travis.yml

```diff
@@ -9,6 +9,7 @@ script:
   - mkdir build
   - cd build
   - cmake ..
+  - make
   - ctest -D ExperimentalBuild
```

Comment:
my (flawed!) undestanding is that the ctest -D ExperimentalBuild does this for me in this case -- what am I missing? :)

The whole ctest thing I only brought in to do the valgrind stuff; I don't really want all that...ideas welcomed!